tests/unit/utils/arrays.spec.js

```diff
@@ -18,4 +18,37 @@ describe('arrays', () => {
       expect(last([])).toEqual(undefined);
     });
   });
+
+  describe('isEqual', () => {
+    const first = [1, 2, 3];
+    const second = [1, 2];
+    const firstStrings = ['1', '2', '3'];
+
+    it('returns true, if passed the same array', () => {
+      expect(isEqual(first, first)).toBe(true);
+    });
+
+    it('returns true, if passed an array, that looks similar', () => {
+      expect(isEqual(first, first.slice(0))).toBe(true);
+    });
+
+    it('returns false, if the two arrays are different', () => {
+      expect(isEqual(first, second)).toBe(false);
+    });
+
+    it('returns false, if the array does not have the same item types', () => {
+      expect(isEqual(first, firstStrings)).toBe(false);
+    });
+
+    it('does not error out, if passed undefined', () => {
```

Comment:
I think you could take this out since I don't think this would ever be an expected way to call this function (with only a single array).